newswires/client/src/App.tsx

```diff
@@ -238,13 +240,25 @@ export function App() {
 								</EuiHeaderSectionItem>
 							</EuiHeaderSection>
 
+							<EuiHeaderSectionItem>
+								<EuiShowFor sizes={['l', 'xl']}>
+									<SearchBox width={'580px'} />
```

Comment:
is there a reason why we couldn't control the width by wrapping `<SearchBox>` in a container? It doesn't feel quite right to me to add a new prop for just the one case


e.g. 

```
<div
	css={css`
		width: 580px;
	`}
>
	<SearchBox/>
</div>
```